controller/src/main/java/org/apache/rocketmq/controller/BrokerHeartbeatManager.java

```diff
@@ -63,4 +64,10 @@ void onBrokerHeartbeat(final String clusterName, final String brokerName, final
      * Check whether broker active
      */
     boolean isBrokerActive(final String clusterName, final String brokerName, final Long brokerId);
+
+    /**
+     * Count the number of active brokers in each broker-set of each cluster
+     * @return active brokers count
+     */
+    Map<String/*cluster*/, Map<String/*broker-set*/, Long/*active broker num*/>> getActiveBrokersNum();
```

Comment:
It's unnecessary to use type `Long` for the number of brokers, `Integer` is enough.